manifests/input.yaml

```diff
@@ -651,28 +661,83 @@ work/input/make_ryf/make_ryf.py:
   hashes:
     binhash: 9fbb6a9f6bec50ccb2afafecae9a5ff7
     md5: c5ef2aeff9516ec76c80e76aef08d352
+work/input/mask_table.464.48x40:
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/mom/mask_table.464.48x40
+  hashes:
+    binhash: 339b390c248283a1bc8113dadf3d4242
+    md5: 7a2fcdfbb9b331161c7d8874e2b7e6f9
+work/input/meshes/JRA55do-ESMFmesh.nc:
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/share/meshes/JRA55do-ESMFmesh.nc
+  hashes:
+    binhash: 3654d5a1e59bff0fe76104c7abaa0b10
+    md5: 1c8481b8c85f59b6cbfc2ed5bc24c77d
+work/input/monthly_sstsss.nc:
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/cice/monthly_sstsss.nc
+  hashes:
+    binhash: 2f705ff1c81531a8da76174241fb0401
+    md5: 598c7aa8a08fe3f48a621f781dd26331
+work/input/o2i.nc:
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/cice/o2i.nc
```

Comment:
also pls delete

Reply:
Update input and restart yaml files in this commit https://github.com/COSIMA/MOM6-CICE6/commit/058e5f7ee3ae4fd031bc1e9f87ae1e74ad263382